lib/components/AnimatedRegion.js

```diff
@@ -119,80 +129,33 @@ export default class AnimatedMapRegion extends AnimatedWithChildren {
   }
 
   spring(config) {
-    var animations = [];
-    config.hasOwnProperty('latitude') &&
-      animations.push(
-        Animated.spring(this.latitude, {
-          ...config,
-          toValue: config.latitude,
-        })
-      );
-
-    config.hasOwnProperty('longitude') &&
-      animations.push(
-        Animated.spring(this.longitude, {
-          ...config,
-          toValue: config.longitude,
-        })
-      );
-
-    config.hasOwnProperty('latitudeDelta') &&
-      animations.push(
-        Animated.spring(this.latitudeDelta, {
-          ...config,
-          toValue: config.latitudeDelta,
-        })
-      );
-
-    config.hasOwnProperty('longitudeDelta') &&
-      animations.push(
-        Animated.spring(this.longitudeDelta, {
-          ...config,
-          toValue: config.longitudeDelta,
-        })
-      );
-
+    const animations = [];
+    for (const type of configTypes) {
+      if (config.hasOwnProperty(type)) {
+        animations.push(
+          Animated.spring(this[type], {
+            ...config,
+            toValue: config[type],
```

Comment:
There's been some reports about a lot of warnings for `useNativeDriver` and am wondering if we should add it here as well.

Reply:
I can throw that in. First, though, I'll search open issues to document `useNativeDriver` reports and ensure there aren't other places we may need to explicitly pass it where, currently, no configuration for the property exists.

Reply:
@christopherdro Per the types and interfaces declared in `index.d.ts`, I think the `Animated.timing` and Animated.spring` configurations with `useNativeDriver: true` should work, and thus eliminate some in-dev warnings developers see that clog up the Flipper <em>et al</em> dialogs. There may be other opportunities, but I think we can safely wait and see if anyone reports additional excessive warnings. See [this commit](https://github.com/react-native-maps/react-native-maps/pull/3744/commits/64a26e05932fc4c5b141f2f77155dfd39088873e).

Reply:
I'm a bit nervous about setting the native driver to true here. What if we expose `useNativeDriver` as a prop on config and default it to false? This way we retain the same behavior, reduce the yellow box warnings and expose an option to opt into using the native driver to test perf improvements.

Reply:
OK -- working on that over the next few days, including updating docs.

Reply:
@christopherdro I opted to pass it directly into the `AnimatedRegion` constructor. Also, I merged `master` in, and it looks like there are some `android` files that weren't part of the original `master` checkout I was working with. Thoughts?

Reply:
Good call. I like the latter where it is part of the param passed into `timing` or `spring`.

Reply:
@jasonmichaels Any updates here?

Reply:
@christopherdro Yeah, sorry about the delay here. Started a new gig back in late March, and been somewhat slammed. Kept meaning to jump back onto this. I just pushed [this commit](https://github.com/react-native-maps/react-native-maps/pull/3744/commits/7627ca353fc8a0356cbf3c71a470832478694406), which coerces it from `config.useNativeDriver`. I'll update the docs next.

Reply:
No worries. I appreciate you taking the time to follow up. Changes look good. I'll give a test over the weekend.